Android Development/2020-2021/Session 1.md

```diff
@@ -0,0 +1,33 @@
+<!-- 
+ - Replace all the {} with their values. Name this file as Session {Session no.}.md (without braces) and submit a PR.
+-->
+# Session 1
+Introduction to Constraint Layout
+Conducted on: 27.01.2020
+
+## Agenda
+An Introduction to Constraint Layout
+
+## Summary
+Constraint Layout is better performance wise because it avoids helps view hierarchy. Flat layouts are quicker to render.
+Various constraints available to create flat complex layouts using Constraint Layout was taught.
+
+Disscussion included:
```

Comment:
check spelling

Reply:
cross check through grammarly too...

Reply:
Thanks for pointing it out, corrected spelling errors.